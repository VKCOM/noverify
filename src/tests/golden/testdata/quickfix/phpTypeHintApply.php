<?php
declare(strict_types = 1);

class SimpleClass
{

/**
 * This is what the variable does. The var line contains the type stored in this variable.
 * @var string
 */
private string $foo;


/**
 * This is what the variable does. The var line contains the type stored in this variable.
 * @var string
 */
private string $foo2 ;

/**
 * @param $str string
 * @param $str2 string
 */
function test($str, $str2){
}
}

/**
 * @param $str string
 */
function test2($str){
}