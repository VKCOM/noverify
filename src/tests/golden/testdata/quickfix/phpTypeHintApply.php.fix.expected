<?php
declare(strict_types = 1);

class SimpleClass
{

/**
 * This is what the variable does. The var line contains the type stored in this variable.
 * @var string
 */
private string $foo;


/**
 * This is what the variable does. The var line contains the type stored in this variable.
 * @var string
 */
private string $foo2 ;

/**
 * @param $str string
 * @param $str2 string
 */
function test(string $str, string $str){
}
}

/**
 * @param $str string
 */
function test2(string $str){
}